### PR TITLE
Update fork

### PR DESCRIPTION
**Pull request by Nicoretti**
* Adjust paths and names
* Remove outdated script and names

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -1,17 +1,21 @@
-name: Run pytest tests
+name: Tests
 
 on:
   push:
     branches-ignore:
-      - github-pages/*
+      - 'main'
```
Comment (tkilias) [inline .github/workflows/tests.yaml:6]: why not on main?

File: .github/workflows/check_documentation_build.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ jobs:
         git config --local user.email "opensource@exasol.com"
         git config --local user.name "GitHub Action"
         git fetch
-        poetry run python -m nox -s push-pages-current
+        poetry run python -m nox -v -s push-pages-current
```
Comment (tkilias) [inline .github/workflows/check_documentation_build.yaml:37]: use long form for  `-v` if available, could find the doc for this parameter

File: .github/workflows/release_droid_upload_github_release_assets.yml [hunk 3]
```diff
@@ -1,86 +0,0 @@
-name: Release Droid - Upload GitHub Release Assets
```
Comment (tkilias) [inline .github/workflows/release_droid_upload_github_release_assets.yml:1]: why did you remove this? we need this still for release-droid?

**Review by tkilias — CHANGES_REQUESTED** [review]

File: .github/workflows/check_documentation_build.yaml (hunk 2 above)
Comment (Nicoretti) [reply .github/workflows/check_documentation_build.yaml:37]: @tkilias long option definitely would make sense, it's a left over though.
Good catch in any case.

```
  -v, --verbose         Logs the output of all commands run including commands marked silent.
```

File: .github/workflows/release_droid_upload_github_release_assets.yml (hunk 3 above)
Comment (Nicoretti) [reply .github/workflows/release_droid_upload_github_release_assets.yml:1]: I will add release related workflows etc. while addressing #3 .

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (Nicoretti) [reply .github/workflows/tests.yaml:6]: Good point, this is a leftover. I wanted to refactor more but reverted it then and decided to address it later.

File: scripts/version_check.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    version = result.stdout.decode().split()[1]
+    return version_from_string(version)
+
+
+def version_from_changelog(path: Path) -> Version:
+    with open(path) as changelog:
+        content = changelog.read()
+        # Search for the FIRST pattern like: "* [0.5.0](changes_0.5.0.md)" in the changelog file.
+        # Note that we encapsulate the [(0.5.0)] with parenthesis, which tells re to return the matching string as group
+        version_match = re.search(r"\* \[([0-9]+.[0-9]+.[0-9]+)]\(\S+\)", content)
```
Comment (tkilias) [inline scripts/version_check.py:81]: Should we also check the version in the changes file?

File: scripts/version_check.py [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+    if not module_version == poetry_version:
+        print(
+            f"Version in pyproject.toml {poetry_version} and {args.version_module} {module_version} do not match!"
+        )
+        if args.fix:
+            print(
+                f"Updating version in file ({args.version_module}) from {module_version} to {poetry_version}"
+            )
+            return _SUCCESS
```
Comment (tkilias) [inline scripts/version_check.py:134]: I think, in the conda-vulnerbility-scanner we had also a check, if the current version is higher than the last tag

**Review by tkilias — CHANGES_REQUESTED** [review]

File: scripts/version_check.py (hunk 4 above)
Comment (Nicoretti) [reply scripts/version_check.py:81]: Maybe later. I would first like to or discuss/decide on how we generally include the change log into the sphinx based documentation, e.g.:

* using `.rst` format for change log files
* include md files in sphinx
* like in the sqla project (with a single `changelog.rst` because sphinx will provide structure anyway)

File: scripts/version_check.py (hunk 5 above)
Comment (Nicoretti) [reply scripts/version_check.py:134]: I think adding checks for tags would make sense in the context of the release workflow.
(noted, also for the generalized version module/script)

**Review by tkilias — APPROVED** [review]
